### DISCUSSION TITLE
What is your average amount of sleep?  あなたの平均睡眠時間を教えてください。

### DISCUSSION THREAD

**Discussion by shintani-y** [NAIST-SE/AttractingContributors#439] [General]
En)What is your average amount of sleep?

Ja)あなたの平均睡眠時間を教えてください。

My fork is here:
https://github.com/shintani-y/AttractingContributors-Average_sleep_time_list

**Comment by aru1702** [comment]
Maybe around 6 hours, or less?
How about you?

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407

**Comment by XC1996114514** [comment]
It depends on the homework but usually I will have 8 hours

So could you help me to filled the readme?
Here is my repository;
https://github.com/XianChen1996/AttractingContributors_type_in_your_favourite_footballteam

Thanks a lot for helping me!
